source/heating_model/constant_tidal_heating.cc

```diff
@@ -0,0 +1,126 @@
+/*
+  Copyright (C) 2011 - 2024 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file LICENSE.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+#include <aspect/heating_model/constant_tidal_heating.h>
+
+
+namespace aspect
+{
+  namespace HeatingModel
+  {
+    template <int dim>
+    void
+    ConstantTidalHeating<dim>::
+    evaluate (const MaterialModel::MaterialModelInputs<dim> &/*material_model_inputs*/,
+              const MaterialModel::MaterialModelOutputs<dim> &/*material_model_outputs*/,
+              HeatingModel::HeatingModelOutputs &heating_model_outputs) const
+    {
+      for (unsigned int q=0; q<heating_model_outputs.heating_source_terms.size(); ++q)
+        {
+          // return a constant value
+          heating_model_outputs.heating_source_terms[q] = 2 * constant_viscosity * pow(constant_tidal_strain_rate, 2.0)
+                                                          / ( 1 + pow( (constant_tidal_frequency * constant_viscosity) / constant_shear_modulus, 2.0 ) );
+          heating_model_outputs.lhs_latent_heat_terms[q] = 0.0;
+        }
+    }
+
+
+
+    template <int dim>
+    MaterialModel::MaterialProperties::Property
+    ConstantTidalHeating<dim>::
+    get_required_properties () const
+    {
+      return MaterialModel::MaterialProperties::none;
+    }
+
+
+
+    template <int dim>
+    void
+    ConstantTidalHeating<dim>::declare_parameters (ParameterHandler &prm)
+    {
+      prm.enter_subsection("Heating model");
+      {
+        prm.enter_subsection("Constant tidal heating");
+        {
+          prm.declare_entry ("Constant tidal frequency", "2e-5",
+                             Patterns::Double (0),
+                             "The tidal frequency of moon. "
+                             "Reference value is for Europa, ~3.55 days. "
```

Comment:
I can't quite connect these two values. When I compute `1 / (3.55 * 24*3600) * 2 * pi`, I get something substantially different, namely 0.000028. What is the disconnect here in my calculation?